### PR TITLE
Feat : Added producrt details and added Products .

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
Added :
1) added Details of all the products .
2) Added All the products .

### CODE REVIEW

File: src/components/Products/Products.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+function Products() {
+  const [product, setProduct] = useState([]);
+
+  const getProducts = async () => {
+    try {
+      const { data } = await axios.get("/api/products");
+      setProduct(data.products);
+      console.log(data);
+    } catch (error) {
+      console.log(error);
```
Comment (ozairkhan2810) [inline src/components/Products/Products.jsx:15]: remove console

File: src/components/Products/Products.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                  } else if (badge === "Bestseller") {
+                    return <div class="product--badge--yellow">{badge}</div>;
+                  } else {
+                    return <div class="product--badge--red">{badge}</div>;
+                  }
+                })()}
+              <img src={img} alt="img" />
+              <div class="product--rating">
+                {[...Array(rating)].map(() => (
+                  <AiFillStar size={13} />
```
Comment (ozairkhan2810) [inline src/components/Products/Products.jsx:40]: This is nicely done

**Review by ozairkhan2810 — APPROVED** [review]
everything looks good, just remove console logs

File: src/components/Utils/GetProducts.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const [product, setProduct] = useState([]);
+
+const GetProducts = async () => {
+  try {
+    const { data } = await axios.get("/api/products");
+    setProduct(data.products);
+    console.log(data);
+  } catch (error) {
+    console.log(error);
+  }
```
Comment (sushil-mod) [inline src/components/Utils/GetProducts.js:13]: same console logs

**Review by sushil-mod — APPROVED** [review]
good code ,nice you have destructured  data directly from response .

**Pull request merged**